client/dashboard/profile-wizard/steps/theme/index.js

```diff
@@ -78,48 +63,6 @@ class Theme extends Component {
 		/* eslint-enable react/no-did-update-set-state */
 	}
 
-	getProductIds() {
-		const productIds = [];
-		const profileItems = get( this.props, 'profileItems', {} );
-
-		profileItems.product_types.forEach( product_type => {
-			if (
-				wcSettings.onboarding.productTypes[ product_type ] &&
-				wcSettings.onboarding.productTypes[ product_type ].product
-			) {
-				productIds.push( wcSettings.onboarding.productTypes[ product_type ].product );
-			}
-		} );
-
-		const theme = wcSettings.onboarding.themes.find(
-			themeData => themeData.slug === profileItems.theme
-		);
-
-		if ( theme && theme.id && ! theme.is_installed ) {
-			productIds.push( theme.id );
-		}
-
-		return productIds;
-	}
-
-	redirectToCart() {
-		document.body.classList.add( 'woocommerce-admin-is-loading' );
-
-		const productIds = this.getProductIds();
-		const backUrl = getAdminLink( getNewPath( {}, '/', {} ) );
-		const { connectNonce } = getSetting( 'onboarding', {} );
-
-		const url = addQueryArgs( 'https://woocommerce.com/cart', {
-			'wccom-site': getSetting( 'siteUrl' ),
-			'wccom-woo-version': getSetting( 'wcVersion' ),
-			'wccom-back': backUrl,
-			'wccom-replace-with': productIds.join( ',' ),
-			'wccom-connect-nonce': connectNonce,
```

Comment:
Oops, I didn't realize that we had competing code loaded here and in the dashboard.

The dashboard implementation seems to be missing a few of these parameters needed for the in-app purchase flow:

https://github.com/woocommerce/woocommerce-admin/blob/cc3849d05879ad08efa4b1e6817e27f57f732b3b/client/dashboard/index.js#L72-L75

Reply:
>Oops, I didn't realize that we had competing code loaded here and in the dashboard.

Neither did I 😄 

>The dashboard implementation seems to be missing a few of these parameters needed for the in-app purchase flow:

Added these back in https://github.com/woocommerce/woocommerce-admin/pull/3340/commits/e83b8b72b0c578c3d3eb6db5c6fd7043e60f2612